.github/workflows/ci.yml

```diff
@@ -61,8 +61,6 @@ jobs:
       matrix:
         env:
           - TEST: pulp
-          - TEST: docs
-          - TEST: s3
```

Comment:
Why are those removed?

Reply:
Because `test_s3` and `docs_test` are `false` in your template_config.yml. There was a bug that got fixed where these tests weren't being properly disabled.